src/Schema/Events/Click.ts

```diff
@@ -1869,3 +1869,192 @@ export interface ClickedAlertsFilters {
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+  artwork_id: string
+  price: number
+  last_offer_sent: number
+  new_saves: number
+  page: number
+}
+
+/**
+ * A user clicks on Add Missing Artworks Details on the Send Offers page in CMS
+ *
+ * This schema describes events sent to Segment from [[ClickedAddMissingArtworksDetails]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedAddMissingArtworksDetails",
+ *    context_module: "Send offers",
+ *    context_page_owner_type: "sendOffers",
+ *    context_page_owner_id: "",
+ *    context_page_owner_slug: "",
+ *    partner_id: "61bcda16515b038ce5000104", 
+ *    arwork_id: "60de173a47476c000fd5c4cc",
+ *    price: $20,000,
+ *    last_offer_sent: 20,
+ *    new_saves: 5,
+ *    page: 1
+ *  }
+ * ```
+ */
+ export interface ClickedAddMissingArtworksDetails {
+  action: ActionType.clickedAddMissingArtworksDetails
+  context_page_module: string
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+  artwork_id: string
+  price: number
+  last_offer_sent: number
+  new_saves: number
+  page: number
+}
+
+
+/**
+ * A user clicks on pagination on the Send Offers page in CMS
+ *
+ * This schema describes events sent to Segment from [[ClickedOnPagination]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOnPagination",
+ *    context_module: "Send offers",
+ *    context_page_owner_type: "sendOffers",
+ *    context_page_owner_id: "",
+ *    context_page_owner_slug: "",
+ *    partner_id: "61bcda16515b038ce5000104", 
+ *    label: "1",
+ *    current_page: 1,
+ *    next_page: 2
+ *  }
+ * ```
+ */
+ export interface ClickedOnPagination {
+  action: ActionType.clickedOnPagination
+  context_page_module: string
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+  label: string
+  current_page: number
+  next_page: number
+}
+
+/**
+ * A user clicks on Send Offers to Collectors in the modal for partner offers.
+ *
+ * This schema describes events sent to Segment from [[ClickedSendOfferCollectors]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedSendOfferCollectors",
+ *    context_module: "Send offers",
+ *    context_page_owner_type: "sendOffers",
+ *    context_page_owner_id: "",
+ *    context_page_owner_slug: "",
+ *    partner_id: "61bcda16515b038ce5000104", 
+ *    arwork_id: "60de173a47476c000fd5c4cc",
+ *    price: $20,000,
+ *    collectors: 4
+ *  }
+ * ```
+ */
+ export interface ClickedSendOfferCollectors {
+  action: ActionType.clickedSendOfferCollectors
+  context_page_module: string
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+  artwork_id: string
+  price: number
+  collectors: number
+}
+
+/**
+ * A user clicks on Send Offers to Collectors in the modal for partner offers.
```

Comment:
This description is a duplicate of the description for `ClickedSendOfferCollectors`. Which icon does this refer to?

Reply:
I was thinking about our discussion if partners are clicking on ❤️ or the collector icon, but I substitute by ClickedUpdateArtwork (your next comment)

Reply:
I remember our discussion about tracking clicks on row elements, but I've had doubts about how valuable that data would be, so I think this is a good change.

As a side note, one of the things we wanted to know is whether users would try to click on the artwork thumbnail or title to navigate to the artwork page. We've since decided to link those (and the artist name).